showcases/ui5-tsapp/tsconfig.json

```diff
@@ -8,14 +8,14 @@
     "strict": true,
     "strictPropertyInitialization": false,
     "rootDir": "./webapp",
-    "baseUrl": "./",
     "sourceMap": true,
     "paths": {
       "ui5/ecosystem/demo/tsapp/*": ["./webapp/*"],
-      "ui5/ecosystem/demo/tslib/*": ["./node_modules/ui5-tslib/src/ui5/ecosystem/demo/tslib/*"],
       "unit/*": ["./webapp/test/unit/*"],
-      "integration/*": ["./webapp/test/integration/*"]
-    }
+      "integration/*": ["./webapp/test/integration/*"],
+      "ui5/ecosystem/demo/tslib/*": ["./node_modules/ui5-tslib/src/ui5/ecosystem/demo/tslib/*"]
```

Comment:
What' the difference here? Why do we still need a path into node_modules?

Reply:
Or is this related to the following statement from the [issue report](https://github.com/SAP/ui5-typescript/issues/382#issuecomment-1517501426)?? 

> "The paths mapping is just included in case of the library hasn't been built yet in the monorepo structure. But this mapping can also be omitted. "

Reply:
The paths mappings are still necessary as long as the library has not been built. Then this is used to lookup the type information from the `ts` files. Once built, the type definitions are retrieved from the `dist` folder, then the `index.d.ts` and the nested `d.ts` files are used and the `typeRoots` configuration wins.